app/pods/document/controller.js

```diff
@@ -0,0 +1,19 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+
+export default class DocumentController extends Controller {
+  @service router;
+  @tracked transition;
+
+  @action
+  transitionBack() {
+    // If no route where you returned from go to the home page
```

Comment:
Ik vraag me af of `history.back()` hier voorlopig niet voldoende zou zijn ipv de constructie met het doorgeven van  de transition. Wat denk jij @erikap ?

Reply:
Goed voor mij. Ik was daarnaar aan het zoeken in de Ember docs en vond het daar niet, maar dat hoort tot de [browser API](https://developer.mozilla.org/en-US/docs/Web/API/History/back) :blush: